deploy/single_user_deploy.sh

```diff
@@ -140,7 +140,7 @@ usage() {
 }
 
 check_dependencies() {
-    local dependencies=("jq" "mpssh" "ssh" "ldd" "nohup" "pkill" "readlink" "realpath")
+    local dependencies=("jq" "pssh" "ssh" "ldd" "nohup" "pkill" "readlink" "realpath")
```

Comment:
I do not see command `pssh`. The binary coming with the package `pssh` is `parallel-ssh` to me.